doc/manual/configuration.rst

```diff
@@ -1291,6 +1291,11 @@ The following settings are supported:
 |             |                 | At the dependency both names will refer to the same |
 |             |                 | tool.                                               |
 +-------------+-----------------+-----------------------------------------------------+
+| clean       | Boolean         | Drop all the environment and tools collected so far |
```

Comment:
Could we invert the attribute to `inherit`? That would normally be `true`, and the user can set it to false. The idea is that this could actually also be a list some day, e.g. `[environment, tools, sandbox]`. I wouldn't implement this right now but when we name it `inherit` it's natural that we can enumerate in the future what exactly is inherited.

Reply:
:+1: `inherit` sounds good to me. We also had this list idea but we didn't find a usecase for cleaning only one of these.